### DISCUSSION TITLE
BDN XML without PNGs

### DISCUSSION THREAD

**Discussion by edumj** [SubtitleEdit/subtitleedit#7025] [Ideas]
Hi,

can you add the **BDN XML** format for Blu-ray subtitles, but without PNGs? It's like **BDN XML** but with text subtitles, not PGs.
I also tried the TextST export option but doesn't work for **Scenarist BD**, I need just the .xml but instead of "Graphic" tags, with "Text" tags and the correct format for text subtitles. I tried to make a custom format template but still get Parse errors in **Scenarist BD**.

`Error : Length of Text element value is an odd number.
`

or an unknown error adding just 1 character..

I need to know the exact **Sony BDN XML** format for text subtitles.

I need text subtitles to import them as Subpath in **Scenarist BD** for a Playlist, not for every clip.

Thanks!

**Comment by niksedk** [comment]
Could you attach a sample file? (Github accepts zip files)

**Comment by edumj** [comment]
Well, I have this zip:

[Subs.zip](https://github.com/SubtitleEdit/subtitleedit/files/11795040/Subs.zip)


It contains several files:

· **Subs.srt** -> Just a .srt sample with only 2 subs

· **BDN_Index (25 fps).xml** -> the result of exporting the srt to "BDN xml/png" with **Subtitle Edit**

· **Subs.textst** -> the result of exporting srt to "blu-ray TextST" with **Subtitle Edit**, but **Scenarist** won't accept it (it shoud be .tes+.mui).

· **Subs.tes + Subs.tes.mui** -> The result of encoding 2 subtitules manually in **Scenarist BD**.

· **Scenarist+Subtitle Edit.jpg** -> A screen capture with the same parameters in **Scenarits BD** and **Subtitle Edit**

· **Countdown_Oscars.xml** -> The xml I made from the **Subtitle Edit** xml but changing and adding Elements and Attributes based on **Scenarist** errors, but still something is wrong!

The new option should be like "BDN xml/png" but with only the xml and the subtitles inside (I supose the Text tags should contain texts and not links to files... I hope!)

**Comment by edumj** [comment]
By the way, in "Blu-ray TextST" default settings, the "Text box vertical pos" is wrong (880) I ithink it should be 0 (or max. 70), because if I try to copy those settings in **Scenarist** it throws an error:

`Error : Property value is not valid.. [Out of range, some values of TextBoxStruct are out of range.]
`
